plugins/woocommerce/src/Internal/Admin/Settings/PaymentsRestController.php

```diff
@@ -73,6 +73,28 @@ public function register_routes( bool $override = false ) {
 			),
 			$override
 		);
+		register_rest_route(
+			$this->route_namespace,
+			'/' . $this->rest_base . '/country',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::EDITABLE,
+					'callback'            => fn( $request ) => $this->run( $request, 'set_country' ),
+					'validation_callback' => 'rest_validate_request_arg',
+					'permission_callback' => fn( $request ) => $this->check_permissions( $request ),
+					'args'                => array(
+						'location' => array(
+							'description'       => __( 'ISO3166 alpha-2 country code. Defaults to WooCommerce\'s base location country.', 'woocommerce' ),
+							'type'              => 'string',
+							'pattern'           => '[a-zA-Z]{2}', // Two alpha characters.
+							'required'          => false,
```

Comment:
I guess it is required - I don't see we are backing it up with some default - aren't we?

Reply:
Indeed, should be fixed in ca8b21679dc153411a78000395b0fb8868804cdc